projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -89,6 +90,47 @@ import { DataTableGhostLoaderComponent } from './ghost-loader/ghost-loader.compo
             >
             </datatable-summary-row>
           }
+          <ng-template
+            ngx-datatable-body-row
+            #bodyRow
+            let-groupRow="row"
+            let-rowIndex="index"
+            let-groupValue="groupValue"
+            let-disabled="disabled"
+            let-group="group"
+          >
+            @let rowItem = groupRow ?? group;
+            <datatable-body-row
+              role="row"
+              tabindex="-1"
+              #rowElement
+              [disabled]="disabled"
+              [isSelected]="selector.getRowSelected(rowItem)"
+              [innerWidth]="innerWidth"
+              [columns]="columns"
+              [rowHeight]="getRowHeight(rowItem)"
+              [row]="rowItem"
+              [group]="groupValue"
+              [rowIndex]="getRowIndex(rowItem)"
+              [expanded]="getRowExpanded(rowItem)"
+              [rowClass]="rowClass"
+              [displayCheck]="!groupRow ? displayCheck : null"
+              [treeStatus]="!groupRow ? $any(group)?.treeStatus : null"
```

Comment:
```suggestion
              [treeStatus]="!groupRow ? $group?.treeStatus : null"
```

Any is no longer needed as treeStatus is included in the row type